code/modules/client/preference_setup/occupation/occupation.dm

```diff
@@ -462,8 +466,9 @@ datum/category_item/player_setup_item/proc/prune_occupation_prefs()
 	if((GLOB.using_map.flags & MAP_HAS_RANK)\
 	   && (!pref.char_rank || !mil_branches.is_spawn_rank(pref.char_branch, pref.char_rank, S)))
 		pref.char_rank = "None"
+	*/
```

Comment:
We should get around to purging this branch stuff one of these days.

Reply:
Yeah, it's not really useful to us. Doesn't seem to be adaptable to our corporatocracy.